cli/pazuzu/commands.go

```diff
@@ -3,31 +3,91 @@ package main
 import (
 	"bufio"
 	"fmt"
-	"log"
-	"os"
-
 	"github.com/urfave/cli"
 	"github.com/zalando-incubator/pazuzu"
+	"log"
+	"os"
 	"regexp"
 	"text/tabwriter"
 )
 
 var cnfGetCmd = cli.Command{
 	Name:  "get",
 	Usage: "Get pazuzu configuration",
-	Action: func(с *cli.Context) error {
-		// log.Print("Getting pazuzu configuration")
-		// return nil
-		return ErrNotImplemented
+	Action: func(c *cli.Context) error {
```

Comment:
It would be great to move an anonymous function behind `Action` to a real function. It increases the readability a lot. You can check build command for inspiration.